cli/test/e2e/package_repository_test.go

```diff
@@ -64,6 +64,36 @@ func TestPackageRepository(t *testing.T) {
 		kubectl.Run([]string{"get", "pkg/pkg.test.carvel.dev.2.0.0"})
 	})
 
+	logger.Section("adding of existing repository", func() {
+		kappCtrl.Run([]string{"package", "repository", "add", "-r", pkgrName, "--url", pkgrURL})
+	})
+
+	logger.Section("adding of existing repository with new url", func() {
```

Comment:
Isn't this same as the update scenario down below?

Reply:
`adding of existing repository with new url` has new repository url, which means this is same as update scenario, but we want to make sure that add command also follows the update logic.